python/src/aiconfig/editor/client/src/components/EditorContainer.tsx

```diff
@@ -562,34 +561,26 @@ export default function EditorContainer({
         }
 
         const promptName = statePrompt.name;
-        const isStream = isStreamingSupported(statePrompt, stateRef.current);
-
-        if (isStream) {
-          await runPromptStreamCallback(promptName, (event) => {
-            if (event.type === "output_chunk") {
-              dispatch({
-                type: "STREAM_OUTPUT_CHUNK",
-                id: promptId,
-                output: event.data,
-              });
-            } else if (event.type === "aiconfig") {
-              dispatch({
-                type: "CONSOLIDATE_AICONFIG",
-                action,
-                config: event.data,
-              });
-            }
-          });
-          return;
-        } else {
-          const serverConfigRes = await runPromptCallback(promptName);
-
-          dispatch({
-            type: "CONSOLIDATE_AICONFIG",
-            action,
-            config: serverConfigRes.aiconfig,
-          });
-        }
+        const enableStreaming = isStreamingSupported(
+          statePrompt!,
+          stateRef.current
+        );
+
+        await runPromptCallback(promptName, enableStreaming, (event) => {
```

Comment:
```suggestion
        const serverConfigRes = await runPromptCallback(promptName, enableStreaming, (event) => {
```

Reply:
Let's handle non-streaming still, in case external implementations don't implement streaming